ravenpy/models/commands.py

```diff
@@ -179,17 +222,215 @@ class GriddedForcingCommand(NamedTuple):
     dim_names_nc: Tuple[str, str, str] = ("x", "y", "t")
     grid_weights: GridWeightsCommand = None
 
+    template = """
+    :GriddedForcing {name}
+        :ForcingType {forcing_type}
+        :FileNameNC {file_name_nc}
+        :VarNameNC {var_name_nc}
+        :DimNamesNC {dim_names_nc}
+        {grid_weights}
+    :EndGriddedForcing
+    """
+
     def to_rv(self):
-        pat = """
-:GriddedForcing {name}
-\t:ForcingType {forcing_type}
-\t:FileNameNC {file_name_nc}
-\t:VarNameNC {var_name_nc}
-\t:DimNamesNC {dim_names_nc}
-{grid_weights}
-:EndGriddedForcing
-        """
-        d = self._asdict()
+        d = asdict(self)
         d["dim_names_nc"] = " ".join(self.dim_names_nc)
-        d["grid_weights"] = self.grid_weights.to_rv()
-        return pat.format(**d)
+        d["grid_weights"] = self.grid_weights
+        return dedent(self.template).format(**d)
+
+
+@dataclass
+class HRUStateVariableTableCommandRecord(RavenConfig):
```

Comment:
That's a mouthful. I understand the desire to keep a consistent syntax, but the risk here is that this doesn't sound like a hydrological model variable at all. I suggest having a copy of this class with a shorter name, e.g. `HRUState`.

Reply:
Done in: https://github.com/CSHS-CWRA/RavenPy/pull/30/commits/5e9ca832c44e8cccc07373ceeedb647edbe825ae

I'm closing this PR to avoid further confusion with the Routing Emu one.